addons/mrp/models/mrp_workorder.py

```diff
@@ -140,12 +140,7 @@ def _read_group_workcenter_id(self, workcenters, domain, order):
     production_date = fields.Datetime('Production Date', related='production_id.date_planned_start', store=True, readonly=False)
     json_popover = fields.Char('Popover Data JSON', compute='_compute_json_popover')
     show_json_popover = fields.Boolean('Show Popover?', compute='_compute_json_popover')
-    consumption = fields.Selection([
-        ('strict', 'Strict'),
-        ('warning', 'Warning'),
-        ('flexible', 'Flexible')],
-        required=True,
-    )
+    consumption = fields.Selection(related='production_id.consumption')
```

Comment:
![](https://media.giphy.com/media/3o6ZsZKbgw4QVWEbzq/giphy.gif)

Reply:
on ne veux pas que la consumption change en cours de production si tu change sur la bom

Reply:
It is the consumption of the workorder (not the one of the production). And the consumption of the production shouldn't change (as you say) then no problem for the workorder